apps/users/views/profile/face.py

```diff
@@ -77,16 +74,14 @@ def form_valid(self, form):
         except (errors.MFAFailedError, errors.BlockMFAError) as e:
             form.add_error('code', e.msg)
             return super().form_invalid(form)
-
-        auth_logout(self.request)
         return super().form_valid(form)
 
     def get_success_url(self):
         message_data = {
-            'title': _('Face recognition disable success'),
-            'message': _('Face recognition disable success, return login page'),
-            'interval': 5,
-            'redirect_url': reverse('authentication:login'),
+            'title': _('Face unbinding successful'),
+            'message': _('Face unbinding successful'),
+            'interval': 2,
+            'redirect_url': '/ui/#/profile/index'
         }
         url = FlashMessageUtil.gen_message_url(message_data)
         return url
```

Comment:
There have been few changes and adjustments made to improve the clarity of the code without introducing any serious issues. The code's purpose is not clear, so I need more information regarding its intended functionality. 

Please provide:

1. A description or context about what problem you're trying to solve with this code snippet.
2. More accurate data types like integers instead of floats since they're appropriate for counts within the Django app framework.
3. Instructions on which fields should be checked if there are any.

I will then review the adjusted code, ensuring it adheres to best practices such as using meaningful variable names, checking the right data types where needed, handling exceptions appropriately, and avoiding repetition across multiple functions/classes.